hyperSpec/vignettes/fileio.Rmd

```diff
@@ -164,7 +164,7 @@ If you need help finding out how to import your data, please search and eventual
 ```{block, type="note", echo=TRUE}
 **Reproducing the Examples in this Manual**
 
-To run the code examples, you should create a folder named "fileio" in your working directory and copy the contents of directory <https://github.com/cbeleites/hyperSpec/tree/develop/Vignettes/fileio>.
+To run the code examples, create a folder named "fileio" in your working directory and copy the contents of the directory <https://github.com/cbeleites/hyperSpec/tree/develop/Vignettes/fileio>.
 This online directory contains the required datasets (via `git-lfs`).
```

Comment:
These lines should be corrected after we get rid of `git-lfs` 

The link https://github.com/cbeleites/hyperSpec/tree/develop/Vignettes/fileio will also get broken in the future.

Reply:
Wait, we still use `git-lfs`?

Reply:
we do

Reply:
Oh right, because of large data sets. We got rid of `make`